crates/propolis-api-types/src/instance_spec/v0/mod.rs

```diff
@@ -113,6 +113,19 @@ pub struct DeviceSpecV0 {
     pub network_devices: HashMap<SpecKey, NetworkDeviceV0>,
     pub serial_ports: HashMap<SpecKey, components::devices::SerialPort>,
     pub pci_pci_bridges: HashMap<SpecKey, components::devices::PciPciBridge>,
+
+    /// This field has a default value (`None`) to allow for
```

Comment:
turbo nit: since this appears in the OpenAPI documentation for the server, I would make these regular comments instead of doc comments, since consumers have no control over our serde inside baseball (and it appears, from the regenerated spec, that the attributes don't actually change any of the semantics of the client)